Common/Orders/Fees/AlphaStreamsFeeModel.cs

```diff
@@ -0,0 +1,163 @@
+            new Dictionary<string, CashAmount> { { Market.USA, new CashAmount(0.4m + 0.1m, "USD") } };
+
+        public AlphaStreamsFeeModel(decimal liborRate = 0.024m)
+        {
+            _liborRate = Math.Abs(liborRate);
+            _equityFee.Add(Market.USA, new EquityFee("USD", 0.004m + _liborRate, 0));
+        }
+        /// <summary>
+        /// Gets the order fee associated with the specified order. This returns the cost
+        /// of the transaction in the account currency
+        /// </summary>
+        /// <param name="parameters">A <see cref="OrderFeeParameters"/> object
+        /// containing the security and order</param>
+        /// <returns>The cost of the order in units of the account currency</returns>
+        public override OrderFee GetOrderFee(OrderFeeParameters parameters)
+        {
+            var order = parameters.Order;
+            var security = parameters.Security;
+
+            // Option exercise for equity options is free of charge
+            if (order.Type == OrderType.OptionExercise)
+            {
+                var optionOrder = (OptionExerciseOrder)order;
+
+                if (optionOrder.Symbol.ID.SecurityType == SecurityType.Option &&
+                    optionOrder.Symbol.ID.Underlying.SecurityType == SecurityType.Equity)
+                {
+                    return OrderFee.Zero;
+                }
+            }
+
+            decimal feeResult;
+            string feeCurrency;
+            var market = security.Symbol.ID.Market;
+            switch (security.Type)
+            {
+                case SecurityType.Forex:
+                    // get the total order value in the account currency
+                    var totalOrderValue = order.GetValue(security);
+                    var fee = Math.Abs(_forexCommissionRate * totalOrderValue);
+                    feeResult = Math.Max(0, fee);
+                    // Forex fees are all in USD
+                    feeCurrency = Currencies.USD;
+                    break;
+
+                case SecurityType.Option:
+                    CashAmount optionsFeeRatePerContract;
+                    if (!_optionFee.TryGetValue(market, out optionsFeeRatePerContract))
+                    {
+                        throw new Exception($"AlphaStreamsFeeModel(): unexpected future Market {market}");
+                    }
+                    feeResult = order.AbsoluteQuantity * optionsFeeRatePerContract.Amount;
+                    feeCurrency = optionsFeeRatePerContract.Currency;
+                    break;
+
+                case SecurityType.Future:
+                    if (market == Market.Globex || market == Market.NYMEX
+                        || market == Market.CBOT || market == Market.ICE
+                        || market == Market.CBOE || market == Market.NSE)
+                    {
+                        // just in case...
+                        market = Market.USA;
```

Comment:
if we're going to do all of this, and since the dictionary has exactly one value, I'm wondering of the benefit of using the dictionary at all -- why not handle it like forex, or conversely, why isn't forex handled like all the others?